sdk/src/asset_handlers/sfnt_io.rs

```diff
@@ -332,13 +334,17 @@ impl SfntFont {
                     // the C2PA table - that's the only one we should ever
                     // be removing.
                     if self.tables.contains_key(&C2PA_TABLE_TAG) {
-                        return Err(wrap_font_err(FontError::SaveError));
+                        return Err(FontError::SaveError(FontSaveError::UnexpectedTable(
+                            format!("{:?}", C2PA_TABLE_TAG),
+                        )));
                     }
                     // We removed exactly one table
                     -(size_of::<SfntDirectoryEntry>() as i64)
                 } else {
                     // We added some other number of tables. Weird, right?
-                    return Err(wrap_font_err(FontError::SaveError));
+                    return Err(FontError::SaveError(
+                        FontSaveError::UnexpectedNumberOfTables,
```

Comment:
I think here instead maybe we have a `TooManyTablesRemoved` error to be more specific